src/test/fuzz/txrequest.cpp

```diff
@@ -147,6 +143,7 @@ class Tester
 
     void AdvanceTime(std::chrono::microseconds offset)
     {
+        assert(offset.count() >= 0);
```

Comment:
Does that mean Bitcoin Core will crash when the user adjusts the time?

Reply:
No, this is in the fuzz test. I wanted to verify that the test wasn't wasting time by trying situations in which time go backwards (which are now pointless).

Reply:
Thanks. Didn't realize this was in the fuzz test. :see_no_evil: